### DISCUSSION TITLE
Splash Screen & Debugging

### DISCUSSION THREAD

**Discussion by ylazy** [airsdk/Adobe-Runtime-Support#2238] [General]
If an error occurs while the splash screen is displaying, the Splash Screen will continue to display forever until I close the debugger. I can't view & edit my code @@!

![image](https://user-images.githubusercontent.com/6010183/198025197-c7d24f24-a7a2-4ee2-aff9-da73aea60f56.png)
![image](https://user-images.githubusercontent.com/6010183/198024998-6db696ff-5f60-4002-a1ae-2c3cbec7c43e.png)

**Comment by ylazy** [comment]
@ajwfrost Please take a look at this. Splash screen shouldn't be displayed during debugging. At least a skip button should be displayed to hide the logo.

![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/6010183/03a777ae-b38e-4e91-b7e4-4981d3e1ce2a)

Thanks!

**Reply by ajwfrost** [reply]
Sorry, I'd missed this when it was first submitted -> we can put a check in to the splash screen code to see if the debugger is active, which should avoid this situation.

thanks
